### PR TITLE
Linux install istructions

### PR DESCRIPTION
**Pull request by PsykeDady**
thank for tool. Is really useful. By the way, i added linux install instruction and i corrected a command usage.

### CODE REVIEW

**Comment by Uzaaft** [comment]
There is an easier way to do this: `cargo install --path . ` after cloning and entering the repository. 
This will add the CLI to `~/.cargo/bin/`

**Comment by PsykeDady** [comment]
But you must have `~/.cargo/bin` in your own PATH. 
I can add as another installation method i suppone

File: README.md [hunk 1]
```diff
@@ -39,11 +40,37 @@ brew install git-ignore
...
+```bash
+sudo mv target/debug/git-ignore /usr/bin/git-ignore
+```
+
 ## Usage 🚀
 
 **git-ignore** is used to generate new `.gitignore` files or append content to existing ones.
 
-From now on, you simply run `git ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
+From now on, you simply run `git-ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
```
Comment (janniks) [inline README.md:73]:
```suggestion
From now on, you simply run `git ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
```

The cool thing with git is, we can actually run this as `git ignore`. Would love to keep this as is.

**Review by janniks — CHANGES_REQUESTED** [review]
Awesome, thank you so much for the detailed instructions

**Comment by Uzaaft** [comment]
> But you must have `~/.cargo/bin` in your own PATH.
> I can add as another installation method i suppone

I think having this as a must is a sensible thing to do, considering the fact that the rust-lang website even mentions this.
Besides, if you were to add it manually, since it is a a user binary, it  should be placed in `/usr/local/binary` and not `/usr/bin`

**Comment by janniks** [comment]
🚀 Thanks for the input @PsykeDady and @Uzaaft — This was very helpful! I expanded the instructions a bit and think it now covers both cases. 🙏

**Comment by PsykeDady** [comment]
Thanks to you!

**Pull request merged**